tests/e2e/utils/RequestHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { TestConstants, RequestType } from '..';
+import axios from 'axios';
+import querystring from 'querystring';
+import { injectable } from 'inversify';
+
+@injectable()
+export class RequestHandler {
+    async processRequest(reqType: RequestType, url: string, data?: string) {
+        let response;
+        // maybe this check can be moved somewhere else at the begining so it will be executed just once
+        if (TestConstants.TS_SELENIUM_MULTIUSER === true) {
+            let authorization = 'Authorization';
+            axios.defaults.headers.common[authorization] = 'Bearer ' + await this.getCheBearerToken();
+        }
+        switch (reqType) {
+            case RequestType.GET: {
```

Comment:
What do you think about creating a dedicated methods:
 - get
 - post
 - delete
 ...
it will allow to get rid of the "RequestType" and additional logic.

Reply:
They will be a duplication of the axios methods but with already added authorization token

Reply:
This makes sense to me as well.